src/cl/nimbus/nimbus_launcher.star

```diff
@@ -374,21 +374,26 @@ def get_beacon_config(
     if node_keystore_files != None and not split_mode_enabled:
         cmd.extend(validator_flags)
 
-    if network == "kurtosis":
-        if bootnode_contexts == None:
-            # Copied from https://github.com/status-im/nimbus-eth2/blob/67ab477a27e358d605e99bffeb67f98d18218eca/scripts/launch_local_testnet.sh#L417
-            # See explanation there
-            cmd.append("--subscribe-all-subnets")
-        else:
-            for ctx in bootnode_contexts[: constants.MAX_ENR_ENTRIES]:
-                cmd.append("--bootstrap-node=" + ctx.enr)
-                cmd.append("--direct-peer=" + ctx.multiaddr)
-    elif network not in constants.PUBLIC_NETWORKS:
+    if network not in constants.PUBLIC_NETWORKS:
         cmd.append(
             "--bootstrap-file="
             + constants.GENESIS_CONFIG_MOUNT_PATH_ON_CONTAINER
             + "/bootstrap_nodes.txt"
         )
+        if network == "kurtosis":  # Kurtosis
+            if bootnode_contexts == None:
+                cmd.append("--subscribe-all-subnets")
+            else:
+                for ctx in bootnode_contexts[: constants.MAX_ENR_ENTRIES]:
+                    cmd.append("--bootstrap-node=" + ctx.enr)
+                    cmd.append("--direct-peer=" + ctx.multiaddr)
+    #     elif network == "ephemery": # Ephemery
```

Comment:
lets not have commented code, lets just delete this

Reply:
I'm dragging my feet with this: https://github.com/kurtosis-tech/ethereum-package/issues/455. Will work on it.

Reply:
We can always add it back, look at Git history, the dead code doesn't feel nice.

Reply:
Removed till 455 is solved.